lib/screens/app/wallet/dashboard.dart

```diff
@@ -329,16 +329,19 @@ class _DashboardState extends State<Dashboard> {
     TransactionType type,
   }) {
     showModalBottomSheet(
+        isScrollControlled: true,
         context: context,
-        elevation: 0,
         shape: RoundedRectangleBorder(
           borderRadius: BorderRadius.only(topLeft: Radius.circular(25), topRight: Radius.circular(25)),
         ),
         builder: (BuildContext context) {
-          return TransactionDialog(
-            transaction: transaction,
-            member: member,
-            transactionType: type,
+          return Container(
```

Comment:
fixed number because it's now a scroll view and needs a height constraint? Trying to understand where the magic number 350 is coming from...

Reply:
I should know better than to have a magic number sorry, Change it for 50% of context bottomSheet